tests/CMakeLists.txt

```diff
@@ -26,19 +26,12 @@ add_subdirectory(utils/stl-lit)
 find_package(Python "3.13" REQUIRED COMPONENTS Interpreter)
 
 if(NOT DEFINED LIT_FLAGS)
-    list(APPEND LIT_FLAGS "-o" "${CMAKE_CURRENT_BINARY_DIR}/test_results.json")
+    set(LIT_FLAGS "-o" "${CMAKE_CURRENT_BINARY_DIR}/test_results.json")
 endif()
 
+set(STL_LIT_COMMAND ${Python_EXECUTABLE} ${STL_LIT_OUTPUT} ${LIT_FLAGS})
 get_property(STL_LIT_TEST_DIRS GLOBAL PROPERTY STL_LIT_TEST_DIRS)
-list(APPEND STL_LIT_COMMAND "${STL_LIT_OUTPUT}"
-                            "${LIT_FLAGS}"
-                            "-D" "notags=ASAN"
-                            "${STL_LIT_TEST_DIRS}")
-list(APPEND STLASAN_LIT_COMMAND "${STL_LIT_OUTPUT}"
-                                "${LIT_FLAGS}"
-                                "-D" "tags=ASAN"
-                                "${STL_LIT_TEST_DIRS}")
-
-add_test(NAME stl COMMAND ${Python_EXECUTABLE} ${STL_LIT_COMMAND} COMMAND_EXPAND_LISTS)
-add_test(NAME stlasan COMMAND ${Python_EXECUTABLE} ${STLASAN_LIT_COMMAND} COMMAND_EXPAND_LISTS)
-set_tests_properties(stl stlasan PROPERTIES RUN_SERIAL ON)
+
+add_custom_target(test COMMAND ${STL_LIT_COMMAND} -Dnotags=ASAN ${STL_LIT_TEST_DIRS} USES_TERMINAL)

```

Comment:
`test` is essentially impossible to search for - it's really, really important to be able to trace through build systems and figure out what links with what. I strongly recommend renaming this to something like `normal-tests` or `vanilla-tests` (for symmetry with `extra-asan-tests`). Then I believe you will need to update:

* `azure-devops/build-and-test.yml`
  + `default: 'test'`
* `README.md`
  + ``the plain `test` target to run the "normal" set of tests``
  + `` `ninja test extra-asan-tests` ``

Reply:
Given that we never use these directly - they exist solely for the CI pipelines - why not simply name them `STL-CI` and `STL-ASan-CI`, and skip mentioning them in the `README` at all?

Reply:
That sounds good to me. We aren't currently using those names in the infra, so they will be searchable.